### PR TITLE
Чирков Р.Д. 381806-2. ЛР2.

### PR DESCRIPTION
**Pull request by ChirkovRoman**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by ChirkovRoman** [comment]
@WolfCub15  @stasyurin проверьте, пожалуйста.

**Review by stasyurin — APPROVED** [review]
👌

File: modules/reverse-polish-notation/test/test_rpn.cpp [hunk 1]
```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Chirkov Roman
+
+#include <gtest/gtest.h>
+#include <include/rpn.h>
+
+#include <string>
+
```
Comment (WolfCub15) [inline modules/reverse-polish-notation/test/test_rpn.cpp:7]: Стандартная библиотека должна идти перед .h файлом Вашего проекта  [пример](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

**Review by WolfCub15 — CHANGES_REQUESTED** [review]

File: modules/reverse-polish-notation/test/test_rpn.cpp (hunk 1 above)
Comment (ChirkovRoman) [reply modules/reverse-polish-notation/test/test_rpn.cpp]: Благодарю. Поменял порядок и скобки

**Review by WolfCub15 — APPROVED** [review]

File: lab-guide/topics.md [hunk 2]
```diff
@@ -78,7 +78,7 @@
 | Пересечения прямой и плоскости в 3D | Логанов А.С. 381806-2 | |
 | Треугольник, расчет основных параметров (задается координатами) |Параничева АВ 381806-1 | |
 | Расчёт площади произвольно заданного многоугольника на плоскости | Алибеков М.Р. 381806-1 | |
-| Обратная польская запись | | |
+| Обратная польская запись |Чирков Р.Д. 381806-2| |
```
Comment (dkurt) [inline lab-guide/topics.md:81]: Please remove this file from PR

**Review by streltsova-yana — APPROVED** [review]

**Pull request merged**